com.ibm.wala.core.java11/src/module-info.java

```diff
@@ -1,5 +0,0 @@
-module com.ibm.wala.core.java11 {
```

Comment:
We can delete this, but I'd like to think about the advantages / disadvantages of adopting modules now that we require JDK 11.  Not for this PR